cli/runtime/apis/cli/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -210,3 +210,19 @@ func (in *PluginDescriptor) DeepCopy() *PluginDescriptor {
 	in.DeepCopyInto(out)
 	return out
 }
+
```

Comment:
For my learning, how does this file get generated?

Reply:
It gets generated with controller-gen. I believe this target generates it. https://github.com/vmware-tanzu/tanzu-framework/blob/main/cli/runtime/Makefile#L73-L76